src/scenic/domains/driving/model.scenic

```diff
@@ -38,6 +44,23 @@ from scenic.domains.driving.behaviors import *
 from scenic.core.distributions import RejectionException
 from scenic.simulators.utils.colors import Color
 
+## 2D mode flag & checks
+
+def is2DMode():
+    from scenic.syntax.veneer import mode2D
+    return mode2D
+
+if 'use2DMap' not in globalParameters:
```

Comment:
You should double-check, but I don't think this conditional is necessary: `param` statements in world models loaded via the `model` statement do not override `param` statements elsewhere. (This is why `scenic.simulators.carla.model` for example can provide a default value for the `weather` parameter without having to worry about whether the user provided a different value before declaring they want to use the CARLA model.)